contract/Makefile

```
@@ -21,6 +21,21 @@ SIGN_BROADCAST_OPTS=--keyring-backend=test --chain-id=$(CHAINID) \
 		--gas=auto --gas-adjustment=$(GAS_ADJUSTMENT) \
 		--yes -b block
 
+mint100:
+	make FUNDS=1000$(ATOM) fund-acct
+	cd /usr/src/agoric-sdk && \
+		yarn --silent agops vaults open --wantMinted 100 --giveCollateral 100 >/tmp/want-ist.json && \
+		yarn --silent agops perf satisfaction --executeOffer /tmp/want-ist.json --from user1 --keyring-backend=test
+
+# https://agoric.explorers.guru/proposal/61
+lower-bundle-cost: scripts/lower-bundle-cost.json ./scripts/voteLatestProposalAndWait.sh
+	agd tx gov submit-proposal param-change scripts/lower-bundle-cost.json \
```

I like the overall approach, but it's difficult to comprehend scripts/lower-bundle-cost.json because there's so much extraneous detail. Suggestion: construct it dynamically to make clear what is changing.
```suggestion
/tmp/lower-bundle-cost.json: .PHONY
	# Read current swingset params JSON,
	# extract beans_per_unit,
	# replace the "storageByte" value per https://agoric.explorers.guru/proposal/61 ,
	# and wrap the result in a proposal.
	agd query swingset params -o json | awk '!in_json { sub(/^[^{]*/, ""); in_json=$$0; } in_json' | \
	jq '.beans_per_unit | map(if .key == "storageByte" then .beans |= "20000000" else . end)' | \
	jq \
	--arg title 'Lower Bundle Cost to 0.02 IST/Kb (a la mainnet 61)' \
	--arg description '0.02 IST/Kb' \
	--arg deposit 10000000ubld \
	--argjson change '{ "subspace": "swingset", "key": "beans_per_unit", "value": "..." }' \
	'. as $$newVal | { title: $$title, description: $$description, deposit: $$deposit, changes: [$$change | .value |= $$newVal] }' \
	> /tmp/lower-bundle-cost.json

lower-bundle-cost: /tmp/lower-bundle-cost.json
	agd tx gov submit-proposal param-change /tmp/lower-bundle-cost.json \
```

---

I'll see what it looks like to do that in js.